Aliases/br

```diff
@@ -0,0 +1 @@
+../Formula/brooklyn-client.rb
```

Comment:
Switchings not ideal, we'd explicitly named to avoid a clash with any commonly used tools and to stay consistent with the likes of `cf`, `aws` etc. Is sticking with the same name as the binary out of the question?